sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -349,7 +351,7 @@ case class SetNamespaceProperties(
  */
 case class SetNamespaceLocation(
     namespace: LogicalPlan,
-    location: String) extends UnaryCommand {
+    location: URI) extends UnaryCommand {
```

Comment:
it's a bit weird to have `URI` in the logical plan, can we keep it as `String` type and just put the qualified path string?

Reply:
and can you refer to the code for qualifying the path string in v1 commands? Let's make sure v1 and v2 are truly consistent.

Reply:
Thanks for catching this. This is where qualified path is generated for v1 catalog (happening in the catalog not in the command): https://github.com/apache/spark/blob/c1029dcff103b1044a55b728c6c98113a255de1e/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala#L285